db/R/dbfiles.R

```diff
@@ -34,16 +34,17 @@
 dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate, mimetype, formatname, parentid=NA, con, hostname=fqdn()) {
   
   name <- basename(in.path)
-  filename <- file.path(in.path,in.prefix)
+  filename <- file.path(in.path, in.prefix)
   
   if (hostname == "localhost") hostname <- fqdn();
   
+  mimetypeid <- db.query(paste0('SELECT id FROM mimetypes where type_string = ', mimetype, ';'))
```

Comment:
what happens if miemtype does not exist? It should then be inserted in the database.